src/app/src/main/kotlin/org/imperial/mrc/hint/db/VersionRepository.kt

```diff
@@ -165,17 +164,38 @@ class JooqVersionRepository(private val dsl: DSLContext) : VersionRepository {
         }
     }
 
-    override fun saveVersionState(versionId: String, projectId: Int, userId: String, state: String)
-    {
+    override fun saveVersionState(versionId: String, projectId: Int, userId: String, state: String) {
         checkVersionExists(versionId, projectId, userId)
         dsl.update(PROJECT_VERSION)
                 .set(PROJECT_VERSION.STATE, state)
                 .where(PROJECT_VERSION.ID.eq(versionId))
                 .execute()
     }
 
-    override fun copyVersion(parentVersionId: String, newVersionId: String, projectId: Int, userId: String)
-    {
+    override fun cloneVersion(parentVersionId: String, newVersionId: String, newProjectId: Int) {
+
+        dsl.insertInto(PROJECT_VERSION,
+                PROJECT_VERSION.PROJECT_ID,
+                PROJECT_VERSION.ID,
+                PROJECT_VERSION.VERSION_NUMBER,
+                PROJECT_VERSION.STATE,
+                PROJECT_VERSION.CREATED,
```

Comment:
I think we have a trigger on the table which will set created and updated on the new row. Not sure that it's desirable for the clone to have the created and updated times anyway.

Reply:
This test suggests that inserting them manually this way works: https://github.com/mrc-ide/hint/pull/334/files#diff-258e0e34eff5999f2b2920095b41713fR344

I would have thought the clone should retain these (especially the created date; updated will change of course if they make any changes.)